IdentityCore/src/network/MSIDURLSessionManager.m

```diff
@@ -25,14 +25,38 @@
 #import "MSIDURLSessionDelegate.h"
 
 static MSIDURLSessionManager *s_defaultManager = nil;
+static NSTimeInterval s_timeoutIntervalForResource = 0;
 
 @implementation MSIDURLSessionManager
 
-+ (void)initialize
+- (instancetype)initWithConfiguration:(NSURLSessionConfiguration *)configuration
```

Comment:
Kai, can you hold this PR for some time? I'm considering let MSAL CPP use this class, which may need some other changes. So MSAL CPP will not need to have its own implementation.

Reply:
Synced offline, for combining CPP and CommonCore logic, we will have a separate item and not rush into the coming Sept hotfix